assets/src/application/ui/display/loadingNotice/types.ts

```diff
@@ -0,0 +1,8 @@
+import { SpinProps, SpinSize } from 'antd/lib/spin';
+
+export interface LoadingNoticeProps {
+	className?: string;
+	loading?: boolean | SpinProps;
```

Comment:
the same approach as here: https://github.com/ant-design/ant-design/blob/master/components/list/index.tsx#L44